### PR TITLE
Implementing new disclosure stage selection

### PR DESCRIPTION
**Pull request by Jordancooke**
- Implemented mock versions of stage 3 and stage 4.
- Updated .gitignore to add node_modules
- Removed the stage specific wranglers.
- Implemented a master disclosure wrangler.
- Updated serverless.yml to reflect changes.
- Deleted wranglers respective test files.

### CODE REVIEW

File: disclosure_wrangler.py [hunk 1]
```diff
@@ -178,4 +233,23 @@ def lambda_handler(event, context):
             return {"success": False, "error": error_message}
 
     logger.info("Successfully completed module: " + current_module)
+
     return {"success": True, "checkpoint": checkpoint}
+
+
+def invoke_method(lambda_execution_name, payload, lambda_client):
```
Comment (kingmushroom) [inline disclosure_wrangler.py:240]: This looks useful enough to make its way to aws_funks in future.

**Review by kingmushroom — APPROVED** [review]
Looks good!

File: README.md [hunk 2]
```diff
@@ -118,6 +97,12 @@ final_output: Dict containing either:<br>
             {"success": True, "data": < stage 2 output - json >}<br>
             {"success": False, "error": < error message - string >}<br>
 
+### Stage 3
+TODO
+
+### Stage 4
+TODO
+
```
Comment (KrisRogos) [inline README.md]: Maybe say "N/A - Currently mocked out" instead, as "TODO" makes it seem like you wanted to implement it but forgot.

File: disclosure_wrangler.py [hunk 3]
```diff
@@ -23,35 +22,42 @@ class EnvironSchema(Schema):
...
+        cell_total_column: The name of the column holding the cell total.
+        top1_column: The name of the column largest contributor to the cell.
+        top2_column: The name of the column second largest contributor to the cell.
+        stage5_threshold: The threshold used in the disclosure calculation.
+        disclosure_stages: The stages of disclosure you wish to run e.g. (1 2 5)
+    }
     :param context: AWS Context Object.
-    :return: Success - True/False & Checkpoint
+    :return final_output: Dict containing either:
+        {"success": True, "data": <stage 2 output - Type: JSON >}
```
Comment (KrisRogos) [inline disclosure_wrangler.py]: Why stage 2?

**Review by KrisRogos — COMMENTED** [review]
Code looks good, just 2 suggestions below regarding documentation/comments

File: disclosure_wrangler.py (hunk 1 above)
Comment (KrisRogos) [reply disclosure_wrangler.py:240]: Make a tech-debt jira ticket for it, looks like we are going to be short on tickets in sprint soon.

File: disclosure_wrangler.py (hunk 3 above)
Comment (Jordancooke) [reply disclosure_wrangler.py]: That was left over from a copy + paste job... I noticed tho that it actually returns a checkpoint and not data so I've now changed it to reflect.

File: README.md (hunk 2 above)
Comment (Jordancooke) [reply README.md]: Updated.

**Review by KrisRogos — APPROVED** [review]

**Pull request merged**